sdks/python/src/opik/url_helpers.py

```diff
@@ -1,8 +1,24 @@
 import opik.config
+import opik.api_objects.opik_client
 
 
 def get_ui_url() -> str:
     config = opik.config.OpikConfig()
     opik_url_override = config.url_override
 
     return opik_url_override.rstrip("/api")
+
+
+def get_experiment_url(dataset_name: str, experiment_id: str) -> str:
```

Comment:
The whole Experiment object could have been passed here since it contains all the necessary fields – dataset_name, client, and experiment ID.
This method could even be made a method of the "Experiment" class itself.

Reply:
@japdubengsub Works as is, we can refactor this in the future if we keep it as is